Changes

```diff
@@ -8,6 +8,26 @@ The changes in each SiliconCompiler release version are described below. Commit
 version shown in (). Where applicable, the contributors that suggested a given
 feature are shown in [].
 
+SiliconCompiler 0.13.2 (2023-08-09)
+=========================================
+
+**Major:**
+
+* Add graphs view to dashboard for comparing metrics across runs.
+
+* Add ``sc-remote`` app for checking server status and interacting with running remote jobs (replaces ``sc-ping``).
+
+**Minor:**
+
+* Add checkbox to dashboard for enabling "raw" view of manifest.
+
+* Tools:
+
+  * OpenROAD - make repair_design more verbose (requires updated version), add option to generate design at end of task, tweak pin placement behavior, write out SDF file in export.
+  * surelog - sanitize escape characters in Verilog output, add disable_write_cache option.
+  * klayout - add option to allow missing cells in stream files, set DB units based on PDK var.
```

Comment:
please add full key path as in for both tools:
 ['tool', 'surelog', 'task', 'parse', 'var', 'enable_lowmem']